test/rekt/resources/deployment/deployment.go

```diff
@@ -34,9 +34,9 @@ var yaml embed.FS
 func Install(name string) feature.StepFn {
 	cfg := map[string]interface{}{
 		"name":      name,
-		"selectors": map[string]string{"app": name},                               // default
-		"image":     "gcr.io/knative-nightly/knative.dev/eventing/cmd/heartbeats", // default
-		"port":      8080,                                                         // default
+		"selectors": map[string]string{"app": name},                                                      // default
+		"image":     "registry.ci.openshift.org/openshift/knative-v1.7:knative-eventing-test-heartbeats", // default
```

Comment:
I'd prefer using the `<name>:<tag>` image, eventually we will drop the other format
```suggestion
		"image":     "registry.ci.openshift.org/openshift/knative-eventing-test-heartbeats:knative-v1.7", // default
```